modules/serverless-apiserversource.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="serverless-apiserversource_context"]
+= Using the API server source
+
+The API server source can be used to connect an event sink (for example, a Knative service) to the Kubernetes API server. The `ApiServerSource` watches for Kubernetes events and forwards them to the Knative Eventing broker.
+
+.Example ApiServerSource YAML
+
+----
+apiVersion: sources.knative.dev/v1alpha1
+kind: ApiServerSource
+metadata:
+  name: testevents
+  namespace: default
+spec:
+  serviceAccountName: events-sa
+  mode: Resource
+  resources:
+    - apiVersion: v1
+      kind: Event
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: Broker
+      name: default
+----
+
+== Prerequisites
+* Using the ApiServerSource requires you to have Knative Serving and Eventing installed.
+// For more information, see the documentation on xref:../.../serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[Installing OpenShift Serverless].
+
+== Procedure
+// . Create the `default` broker in your namespace. For more information, see the documentation on xref:../.../serverless/knative_eventing/serverless-broker.adoc[Creating the broker].
+. Create a Service Account that the `ApiServerSource` runs as. Create a file named `serviceaccount.yaml` and copy the sample code into it:
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
+  namespace: default
```

Comment:
in https://github.com/openshift/openshift-docs/pull/20296/files#diff-e45c2f1575ec9dd1cdc10af23dd4a00aR36  you say in _your namespace_, but in this line you kinda hard-code the namespace to the `default` namespace.

Same on: https://github.com/openshift/openshift-docs/pull/20296/files#diff-e45c2f1575ec9dd1cdc10af23dd4a00aR73

Reply:
I changed "your", but do we need to update the namespaces or anything in the example YAML or is it OK for users to use `default`? We tell users to copy this example into the YAML so it should be something that will work without editing, or else we need to call out what to edit.